db/schema.rb

```diff
@@ -40,10 +40,10 @@
     t.integer "name"
     t.string "level"
     t.integer "years_of_experience"
-    t.integer "volunteer_id"
+    t.integer "user_id"
```

Comment:
I think this should stay as volunteer? - didn't you and Michelle change it back to volunteer in your PR yesterday evening? as the skills just belong to a volunteer

Reply:
So weird, I didn't change that, just pulled from master.

Reply:
haha ok. Yeah I think it should be volunteer, otherwise the seeder won't work